examples/https.jl

```diff
@@ -1,37 +1,24 @@
-"""
-Example of using HttpServer with HTTPS protocol:
-
-1) Create a self-signed SSL Certificate
-    - Generate a private key
-        openssl genrsa -des3 -out server.key 1024
-
-    - Generate a CSR (Certificate Signing Request)
-        openssl req -new -key server.key -out server.csr -subj "/L=github.com/O=JuliaWeb/OU=HttpServer/CN=localhost"
-
-    - Remove Passphrase from Key
-        cp server.key server.key.org
-        openssl rsa -in server.key.org -out server.key
-
-    - Generating a Self-Signed Certificate
-        openssl x509 -req -days 365 -in server.csr -signkey server.key -out server.crt
-
-2) Add certificate/private key pair to a certificate store
+# Generate a certificate and key if they do not exist
+
+rel(p::String) = joinpath(dirname(@__FILE__), p)
+if !isfile("keys/server.crt")
+  @unix_only begin
+    run(`mkdir -p $(rel("keys"))`)
+    run(`openssl req -x509 -nodes -days 365 -newkey rsa:2048 -keyout
```

Comment:
Can you use the mbedtls certificate generation function here instead of the openssl one?